.github/workflows/framework-kind.yml

```diff
@@ -0,0 +1,12 @@
+name: Framework
+
+on:
+  pull_request:
+    branches:
+      - main
+      - release-[0-9]+.[0-9]+
```

Comment:
We don't have "release" branches in this repo, but we do have tags... can this workflow be run when we push a tag like vX.Y.Z?

Reply:
I'm able to enable the workflows on `push`es to the repo. I'm thinking something like `v**` would work?